extensions/spark/kyuubi-spark-connector-hive/src/test/scala/org/apache/kyuubi/spark/connector/hive/HiveQuerySuite.scala

```diff
@@ -33,12 +33,15 @@ class HiveQuerySuite extends KyuubiHiveTest {
     finally spark.sql(s"DROP TABLE $table")
   }
 
-  def withTempPartitionedTable(spark: SparkSession, table: String)(f: => Unit): Unit = {
+  def withTempPartitionedTable(
+      spark: SparkSession,
+      table: String,
+      format: String = "PARQUET")(f: => Unit): Unit = {
     spark.sql(
       s"""
          | CREATE TABLE IF NOT EXISTS
          | $table (id String, year String, month string)
-         | USING PARQUET
+         | USING $format
```

Comment:
I'm not sure how "USING" would be handled in this connector, I verified with `STORED AS` instead

Reply:
Both work and also add unit tests

Reply:
As far as i known, This point depend on the implementation of the catalog. v2HiveCatalog creates tables and queries them, in fact, STORED AS & USING are the same thing.

Reply:
There is a small difference, the format of text does not work when STORED AS is used, only textfile, but USING work well.

Reply:
this is caused by the parsing of spark, In spark, text file format is `text`, hive stored as is `textfile`